### PR TITLE
Fixed some compiler warnings and type issues

### PR DESCRIPTION
**Pull request by clstrfsck**

### CODE REVIEW

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetMessage.groovy [hunk 1]
```diff
@@ -31,7 +31,8 @@ class GetMessage implements Runnable {
     @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
     Composite.IdentityInfo identityInfo
 
-    void run() {
+    @Override
+	void run() {
```
Comment (talekhinezh) [inline radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetMessage.groovy]: tabs

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetDetails.groovy [hunk 2]
```diff
@@ -31,7 +31,8 @@ class GetDetails implements Runnable {
     @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
     Composite.IdentityInfo identityInfo
 
-    void run() {
+    @Override
+	void run() {
```
Comment (talekhinezh) [inline radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetDetails.groovy]: tabs

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetStoredAtoms.groovy [hunk 3]
```diff
@@ -31,7 +31,8 @@ class GetStoredAtoms implements Runnable {
     @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
     Composite.IdentityInfo identityInfo
 
-    void run() {
+    @Override
+	void run() {
```
Comment (talekhinezh) [inline radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetStoredAtoms.groovy]: tabs

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/RadixCLI.groovy [hunk 4]
```diff
@@ -37,7 +37,8 @@ import picocli.CommandLine.Command
         ])
 class RadixCLI implements Runnable {
 
-    void run() {
+    @Override
+	void run() {
```
Comment (talekhinezh) [inline radixdlt-cli/src/main/groovy/com/radixdlt/cli/RadixCLI.groovy]: tabs

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/SendMessage.groovy [hunk 5]
```diff
@@ -39,7 +39,8 @@ class SendMessage implements Runnable {
     @CommandLine.Option(names = ["-d", "--address"], paramLabel = "ADDRESS", description = "Address to which message is sent", required = true)
     String addressString
 
-    void run() {
+    @Override
+	void run() {
```
Comment (talekhinezh) [inline radixdlt-cli/src/main/groovy/com/radixdlt/cli/SendMessage.groovy]: tabs

**Review by talekhinezh — APPROVED** [review]

**Review by siy — APPROVED** [review]
LGTM

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetMessage.groovy (hunk 1 above)
Comment (clstrfsck) [reply radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetMessage.groovy]: Fixed.

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetDetails.groovy (hunk 2 above)
Comment (clstrfsck) [reply radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetDetails.groovy]: Fixed.

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetStoredAtoms.groovy (hunk 3 above)
Comment (clstrfsck) [reply radixdlt-cli/src/main/groovy/com/radixdlt/cli/GetStoredAtoms.groovy]: Fixed.

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/RadixCLI.groovy (hunk 4 above)
Comment (clstrfsck) [reply radixdlt-cli/src/main/groovy/com/radixdlt/cli/RadixCLI.groovy]: Fixed.

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/SendMessage.groovy (hunk 5 above)
Comment (clstrfsck) [reply radixdlt-cli/src/main/groovy/com/radixdlt/cli/SendMessage.groovy]: Fixed.

**Pull request merged**